client/benchmark/build.gradle

```diff
@@ -17,8 +17,7 @@ tasks.named("assemble").configure {  enabled = true }
 archivesBaseName = 'client-benchmarks'
 mainClassName = 'org.elasticsearch.client.benchmark.BenchmarkMain'
 
-// never try to invoke tests on the benchmark project - there aren't any
-tasks.named("test").configure {enabled = false }
+tasks.named("uploadSnykDependencyGraph").configure {enabled = false }
```

Comment:
Maybe this project simply shouldn't have the `elasticsearch.build` plugin applied and could just use `elasticsearch.java` instead?

Reply:
that's what I thought